nrf_security/CMakeLists.txt

```
@@ -55,8 +55,7 @@ if(CONFIG_BUILD_WITH_TFM)
   set_property(TARGET zephyr_property_target
     APPEND PROPERTY TFM_CMAKE_OPTIONS
       -DNRF_SECURITY_SETTINGS=\"$<TARGET_PROPERTY:nrf_security_target,CMAKE_ARGS>\"
-      -DTFM_MBEDCRYPTO_CONFIG_TYPE=STRING
-      -DTFM_MBEDCRYPTO_CONFIG_PATH=$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>
+      -DTFM_MBEDCRYPTO_CONFIG_PATH=secure_fw/partitions/crypto/mbedcrypto/include/$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>
```

What about simply do:
```suggestion
      -DTFM_MBEDCRYPTO_CONFIG_PATH:STRING=$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>
```

---

Why didn't we do it like this in the first place? :D Great solution (y)